codesign/lib/src/globals.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: unnecessary_string_escapes
+
+import 'package:args/args.dart';
+
+class ConductorException implements Exception {
+  ConductorException(this.message);
+
+  final String message;
+
+  @override
+  String toString() => 'Exception: $message';
+}
+
+/// Translate CLI arg names to env variable names.
+///
+/// For example, 'state-file' -> 'STATE_FILE'.
+String fromArgToEnvName(String argName) {
+  return argName.toUpperCase().replaceAll(r'-', r'_');
```

Comment:
Is the raw string necessary?

```suggestion
  return argName.toUpperCase().replaceAll('-', '_');
```

Reply:
Done.